app/services/ExerciseService.ts

```diff
@@ -0,0 +1,90 @@
+import { marked } from 'marked'
+import NotFoundException from '../exceptions/NotFoundException';
+import fs from "fs";
+
+export const ExericseService = {
+
+  getContent(Exercise_ID: string) {
+
```

Comment:
Add documentation for the function also delete the extra line below the function